### DISCUSSION TITLE
Site not published correctly for days

### DISCUSSION THREAD

**Discussion by Tokenyet** [community/community#21680] [Other Feature Feedback, Questions, & Ideas]
<p>I have a site at <a href="https://github.com/bravocareercenter/bravocareercenter.github.io" rel="nofollow noopener">https://github.com/bravocareercenter/bravocareercenter.github.io</a> that shows 404 when i try to visit it through <a href="https://github.com/bravocareercenter/bravocareercenter.github.io" rel="nofollow noopener">bravocareercenter.github.io</a></p>
<p>I know the github site is taked for few days to published in my experience, but this proejct seems to be too long to process, so I ask here for some help to check If I made a mistake or not.</p>

**Comment by SethClydesdale** [comment]
<p>Hi,</p>
<p>Just to confirm: in Settings &gt; Options &gt; Github pages, under Source is your github page currently being built from the master branch?</p>

**Comment by Tokenyet** [comment]
<p>User pages must be built from the master branch.</p>
<p>The <a href="https://github.com/bravocareercenter/bravocareercenter.github.io" title="master branch" rel="nofollow noopener">branch</a> has only one commit (init) and branch (master), so I think It’s correct?</p>

**Comment by Tokenyet** [comment, marked as answer]
<p>Ok, It’s a rare case, I report this issue for github, and the response I got was “Use admin permission to upload the first init”.</p>
<p>So If anyone who push first commit with Write access(co-contributor), you would ask your admin(use admin account) to push the first commit, or commit after first commit with Admin permission.</p>

**Comment by Muntasir2001** [comment]
<p>hi <br>
can u tell me how to do that?</p>
